app/graphql/mutations/events/delete_event.rb

```diff
@@ -1,6 +1,6 @@
 module Mutations
   module Events
-    class DeleteEvent < ::Mutations::BaseMutation
```

Comment:
Discussed yesterday, and these seem unnecessary

Reply:
I'm glad taking this out has workd